### PR TITLE
[winpr,sysinfo] fix GetComputerNameA

### PR DESCRIPTION
**Pull request by akallabeth**
* Trunctate to at most MAX_COMPUTERNAME_LENGTH characters
* Fix usage of function

### CODE REVIEW

File: winpr/libwinpr/sysinfo/sysinfo.c [hunk 1]
```diff
@@ -434,22 +434,18 @@ BOOL GetComputerNameW(LPWSTR lpBuffer, LPDWORD lpnSize)
...
-
 	if (!lpnSize)
 	{
 		SetLastError(ERROR_BAD_ARGUMENTS);
 		return FALSE;
 	}
 
-	if (gethostname(hostname, sizeof(hostname)) == -1)
+	char hostname[256 + 1] = { 0 };
+	if (gethostname(hostname, 256) == -1)
```
Comment (pnowack) [inline winpr/libwinpr/sysinfo/sysinfo.c:444]: Shouldn't it be possible to use something like `ARRAY_SIZE(hostname) - 1` here?

File: winpr/libwinpr/sspi/NTLM/ntlm_av_pairs.c [hunk 2]
```diff
@@ -298,6 +298,30 @@ static BOOL ntlm_av_pair_add_copy(NTLM_AV_PAIR* pAvPairList, size_t cbAvPairList
 	                        ntlm_av_pair_get_value_pointer(pAvPair), (UINT16)avLen);
 }
 
+static char* get_name(COMPUTER_NAME_FORMAT type)
```
Comment (pnowack) [inline winpr/libwinpr/sspi/NTLM/ntlm_av_pairs.c:301]: This function looks to be unused

Comment (akallabeth) [reply winpr/libwinpr/sspi/NTLM/ntlm_av_pairs.c:301]: fixed.

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**